Installing-and-Configuring/Building.md

```diff
@@ -1,15 +1,32 @@
 ---
 layout: default
-title: Building Heads
+title: Step 1 - Building Heads
 permalink: /Building/
-nav_order: 2
+nav_order: 1
 parent: Installing and configuring
 ---
 
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Releases
+===
+
+There are currently no binary downloads; you must build from source to ensure
+ that you can add your own GPG keys to the image to sign your OS installation.
```

Comment:
That is also untruth now for all boards, since the default GUI is gui-init, which will ask the user to inject his public key in rom if none is found. The user also has the capability to generate a key in smartcard, and export the resulting public key from heads, or later on since https://github.com/osresearch/heads/pull/758

Reply:
Actually, you made that section!!! Point to https://github.com/osresearch/heads-wiki/pull/49/files/2ae88c242c581b30dca79e2c1bb3ef3dadfe52f6..99607667c0b26b7f69383d8c956cef44047d631e#diff-c5d3cb596ea06eb42155b5420cbb3120 ?